src/generic_rosenbrock.jl

```diff
@@ -848,6 +869,179 @@ function _transformtab(Alpha,Gamma,B,Bhat)
     (a,C,b,btilde,d,c)
 end
 
+
+
+
+# 3 step ROS Methods
+"""
+    ROS2PRTableau()
+
+2nd order stiffly accurate Rosenbrock-Wanner method with 3 internal stages with (Rinf=0).
+For problems with medium stiffness the convergence behaviour is very poor and it is recommended to use ROS2S instead.
```

Comment:
```suggestion
For problems with medium stiffness the convergence behaviour is very poor and it is recommended to use 
[`ROS2S`](@ref) instead.
```
This adds a reference in the documentation created by Documenter.jl.